profile/profile.yaml

```diff
@@ -0,0 +1,119 @@
+# Preferred name (Characters only)
+name: "Petar Kandic"
+# Your GitHub repo root URL
+github_repo_root_url: https://github.com/jarviscanada/jarvis_data_eng_PetarKandic
+
+# Introduce yourself, your academic and professional background.
+# What excites you about the software industry and your passion. At least 80 words
+summary: For years, I have passionate about both developing software, and learning about the field of software engineering. I began to program as a teenager, focusing on commonly-used languages such as Java. During university, I came to learn more about the theoritical foundations of computer science, and how to apply this knowledge. Finally, I have gained hands-on experience in the professional world. These experiences have allowed me to gain a wide range of skills. My strongest languages are Java and C, though I have experience with other languages, such as Python, JavaScript, and Bash, among others.
+
+# Group your skills by level.
+# Your skills can be anything software related (e.g. language, frameworks, methodologies, software, etc.).
+skills:
+  # At least 5 skills
+  proficient:
+    - Java
+    - Linux/Bash
+    - RDBMS/SQL
+    - Agile/Scrum
+    - Git
+    - C
+  # At least 5 skills
+  competent:
+    - HTML/CSS
+    - JavaScript
+    - Bootstrap
+    - Python
+    - cURL
+    - Debugging
+  # At least 5 skills
+  familiar:
+    - Docker
+    - AWS
+    - Functional Languages/Elixir
+    - C#
+    - C++
+
+# List of Jarvis projects
+jarvis_projects:
+  - name: "Cluster Monitor"
+    git_url: "/linux_sql"
+    # The project description allows hiring managers to understand your project and technologies you used(e.g. programming languages, libraries, algorithms, hardware, tools, etc.).
+    # The project description must start with an action verb (https://bit.ly/3guO98u)
+    description: Developed several applications to monitor the status of the system. Shell scripts are used to gather data on system memory and performance. Then, this is stored in a database.
```

Comment:
make it longer using SDLC. add kesy works, Linux, Shell, Scirpt, PostgreSQL, Docker, Crontab, test, etc..